src/main/scala/com/databricks/labs/overwatch/pipeline/PipelineTargets.scala

```diff
@@ -498,6 +498,22 @@ abstract class PipelineTargets(config: Config) {
       config
     )
 
+    lazy private[overwatch] val notebookCommandsTarget: PipelineTable = PipelineTable(
```

Comment:
you added a target and a module. You need to add them to the Gold target / module inventory.

Gold.scala
`getAllTargets`
`getAllModules`